### ISSUE TITLE
Middle click and scroll don't seem to work

### ISSUE THREAD

**Issue by andrewrota** [facebook/fixed-data-table#24]
Not sure if this is a planned feature or not, but it would be nice to be able to 'autoscroll' with middle-click and scroll as you would in a normal browser window.  Right now it seems like that functionality is unavailable.

**Comment by pieterv** [comment]
Hey @andrewrota, yes unfortunately this feature does not work as we are overriding the browser native scrolling and reimplementing it, this means we have missed off some of these platform dependent features.

@wlis is looking into ways we can take advantage of the browsers native scrolling yet maintain (or increase :P) our scroll perf, which should fix this issue and some of the other related issues.

**Comment by andrewrota** [comment]
Cool, thanks for the quick response!

**Comment by nbostrom** [comment]
We had our own solution before you announced fixed-data-table in which we had a container with overflow: auto. Inside of that container we had another one with a height set to the calculated height of all rows combined. Inside of that we had a absolutely position container with overflow hidden. Inside of that container we rendered the actualt table. The top position always reflected the scroll position of the outer container. This way all of the native scolling aspects worked as expected.

Basically:

    div.scollingContainer {overflow: auto}
        div {height: <actual height of all rows>}
            div {position: absolute, overflow: hidden, height: <height of .scrollingContainer>, top: <scrollTop of .scrollingContainer>}

**Comment by facebook-github-bot-8** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by CASandmann** [comment]
Any news on support for this?  I almost exclusively use the middle mouse click to scroll, so this is a rather frustrating limitation for me.
